jetty-core/jetty-rewrite/src/main/java/org/eclipse/jetty/rewrite/handler/ResponseHeaderRule.java

```diff
@@ -0,0 +1,101 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.rewrite.handler;
+
+import java.io.IOException;
+
+import org.eclipse.jetty.http.HttpStatus;
+import org.eclipse.jetty.server.Response;
+import org.eclipse.jetty.util.Callback;
+import org.eclipse.jetty.util.StringUtil;
+
+/**
+ * Sends the response code whenever the rule finds a match.
+ */
+public class ResponseHeaderRule extends HeaderRule
```

Comment:
Rename to `ResponseStatusHeaderRule`.

Is this rule really needed?
Because it seems to me that `ResponseStatusHeaderRegexRule` can do exactly what this rule can do, and more.

Reply:
No, it's not; my impetus for this was an annoying blackboard "accessibility crawler" that was ignoring my robots.txt and generating a crapload of 400 and 500 errors on a webapp. I put together this one first, and when I went to test it, they had changed their user agent from just "blackboard ally" to also include a version number 8-/. So I sent back and put together another one with a regexp. I considered that this one was a subnet of that one, but I'd already written it and theoretically a direct string comparison is more efficient than a regexp matching a static string ;). I'm fine with deleting it and just submitting the regexp one...

Reply:
I'll just go ahead and whack this one and just submit the regexp based Rule.